bin/init_repo.sh

```diff
@@ -37,12 +40,21 @@ for template_file in "${template_files[@]}" ; do
         for idx in "${!user_values[@]}" ; do
             placeholder=${placeholders[idx]}
             placeholder_name="${placeholder%%:*}"
-            result=`echo "$result" | sed "s/{{$placeholder_name}}/${user_values[idx]}/g"`
+            result=`echo "$result" | sed "s#{{$placeholder_name}}#${user_values[idx]}#g"`
+            if [ "$placeholder_name" = "SRC" ] && [ ${user_values[idx]} != 'src' ]; then
+              mv "$current_path/../src" "$current_path/../${user_values[idx]}"
+            fi
         done
 
-        echo "$result" > "$dest_path"
-        printf "${green}Processed and created ${nc}$dest_path\n"
-        rm "$source_path"
+        if [ -n "${result}" ];
+        then
+          echo "$result" > "$dest_path"
+          arrPath=(${dest_path//\// })
+          mv $dest_path "${current_path}/../${arrPath[${#arrPath[@]}-1]}"
+          printf "${green}Processed and created ${nc}$dest_path\n"
```

Comment:
You aren't printing the real destination path here, but the file path in the `bin` folder before the move.

Reply:
You are right, I'll fix it